src/open_inwoner/scss/views/App.scss

```diff
@@ -195,49 +195,49 @@
   /// Font.
 
   --font-color-heading: var(--color-black);
-  --font-family-heading: 'Heading';
+  // Font-family = 'Heading' coming from both NLDS and/or the font-upload functionality
+  --font-family-heading: var(--utrecht-heading-font-family);
   --font-weight-heading: bold;
 
   --font-color-heading-1: var(--font-color-heading);
   --font-family-heading-1: var(--font-family-heading);
   --font-line-height-heading-1: 41px;
-  --font-size-heading-1: 32px;
+  --font-size-heading-1: var(--utrecht-heading-1-font-size);
   --font-weight-heading-1: var(--font-weight-heading);
 
   --font-color-heading-2: var(--font-color-heading);
   --font-family-heading-2: var(--font-family-heading);
   --font-line-height-heading-2: 34px;
-  --font-size-heading-2: 24px;
+  --font-size-heading-2: var(--utrecht-heading-2-font-size);
   --font-weight-heading-2: var(--font-weight-heading);
 
   --font-color-heading-3: var(--font-color-heading);
   --font-family-heading-3: var(--font-family-heading);
   --font-line-height-heading-3: 30px;
-  --font-size-heading-3: 20px;
+  --font-size-heading-3: var(--utrecht-heading-3-font-size);
   --font-weight-heading-3: var(--font-weight-heading);
 
   --font-color-heading-4: var(--font-color-heading);
   --font-family-heading-4: var(--font-family-heading);
   --font-line-height-heading-4: 26px;
-  --font-size-heading-4: 16px;
+  --font-size-heading-4: var(--utrecht-heading-4-font-size);
   --font-weight-heading-4: var(--font-weight-heading);
 
   --font-size-heading-card: 18px;
 
-  --font-family-body: 'Body';
+  // Font-family = 'Body' coming from both NLDS and/or the font-upload functionality
+  --font-family-body: var(--utrecht-document-font-family);
   --font-color-body: var(--color-gray-dark);
-  --font-size-body: 16px;
+  --font-size-body: var(--utrecht-document-font-size);
   --font-line-height-body: 24px;
 
   --font-size-body-extra-small: 12px;
-  --font-size-body-small: 14px;
+  --font-size-body-small: var(--utrecht-paragraph-small-font-size);
```

Comment:
`--font-size-body` is defined in terms of `--utrecht-document` but `font-size-body-small` is defined in terms of `--utrecht-paragraph`. Is there no `--utrecht-document-small-font-size`?

Reply:
Good for now. If we start using this size elsewhere, we should introduce the modifier as you indicated